time_tracker_jsnjack@gmail.com/extension.js

```diff
@@ -26,7 +26,11 @@ const TTNotificationBanner = new Lang.Class({
     addAction: function(label, callback) {
         // Style buttons
         var extra_style = "";
-        if (label === restart_button_name) {
+        
+        if (label === preferences_button_name) {
+            label = _("Preferences");
```

Comment:
yeah, well.. it should already be in a correct language! I will experiment with it a bit on the weekend. Maybe there is a more elegant way to make it translated